### ISSUE TITLE
@stamp/is version to latest

### ISSUE THREAD

**Issue by kcouliba** [stampit-org/stamp#44]
Hi,

We encountered a conflict using @stamp/it library with @stamp/arg-over-prop, the latest version of @stamp/is currently 1.0.0 should be installed, but version 0.1.2 is installed instead.

**Comment by koresar** [comment]
Could be your `npm` or `yarn` caching is too aggressive for you.

The v0.1.2 you mention does not exist in npm registry.
```
~ npm view @stamp/it

{ name: '@stamp/it',
  description: 'A nice, handy API implementation of the compose standard',
  'dist-tags': { latest: '1.0.1' },
  versions: [ '0.1.4', '0.2.0', '1.0.0', '1.0.1' ],
  time:
   { modified: '2017-07-30T01:30:13.305Z',
     created: '2017-04-14T13:10:20.759Z',
     '0.1.4': '2017-04-14T13:10:20.759Z',
     '0.2.0': '2017-04-22T15:38:17.218Z',
     '1.0.0': '2017-07-29T15:12:55.652Z',
     '1.0.1': '2017-07-30T01:30:13.305Z' },
```

Installation works fine:
```
~ npm i @stamp/it
...
~ cat node_modules/@stamp/it/package.json | grep version
  "version": "1.0.1"
```

Please find the problem on your end or provide more details.

**Comment by koresar** [comment]
Oh, you missed the `@stamp/is` in the message. Thanks! Fixing.

**Comment by koresar** [comment]
Done. Published a patch v1.0.2.

Was
```
~ npm view @stamp/it
...
  'dist-tags': { latest: '1.0.1' },
...
  dependencies:
   { '@stamp/compose': '^1.0.0',
     '@stamp/core': '^1.0.0',
     '@stamp/is': '^0.1.2',            <------ was
     '@stamp/shortcut': '^1.0.1' },
```

Become
```
~ npm view @stamp/it
...
  'dist-tags': { latest: '1.0.2' },
...
  dependencies:
   { '@stamp/compose': '^1.0.0',
     '@stamp/core': '^1.0.0',
     '@stamp/is': '^1.0.0',            <------ become
     '@stamp/shortcut': '^1.0.1' },
```

**Comment by koresar** [comment]
Thank you
